src/StaticCategory/StaticCategory.scss

```diff
@@ -15,4 +15,22 @@
   padding-bottom: 0.5em;
   color: $text-color;
   border-bottom: 0.2em $red solid;
+}
+
+.edit-cat {
+  display: flex;
+  margin-left: 1em;
+  margin-top: 0.5em;
+}
```

Comment:
Whats the difference between edit-cat and edit-category? seems confusing.

Reply:
We made a change to this

Reply:
fixed!